.env.example

```diff
@@ -1,6 +1,8 @@
 CLIENT_ID=
 CLIENT_SECRET=
 
+STAGE_HOSTNAME=
```

Comment:
Would  `ALTERNATIVE_HOSTNAME` be a better name or is it ok that it's so specific?

Reply:
Thank you for the suggestion, yes, much better name! Changed.

Reply:
Nice. Resolving